src/main/java/htsjdk/tribble/util/ParsingUtils.java

```diff
@@ -46,11 +46,10 @@
 
     public static final Map<Object, Color> colorCache = new WeakHashMap<>(100);
 
-    // HTML 4.1 color table,  + orange and magenta
-    static Map<String, String> colorSymbols = new HashMap();
+    private static URLHelperFactory urlHelperFactory = new URLHelperFactory() {};
```

Comment:
It seems strange to have this interface have default implementation of its only method. Lets remove that and just replace this with
```
private static URLHelperFactory urlHelperFactory = RemoteURLHelper::new;
```

Reply:
Sure,  I thought it strange too but didn't want to make a concrete implementation.  I wasn't aware of the syntax you used, nice.

Reply:
Yeah! In java 8+ if you have an interface with exactly 1 abstract method you can use a lambda instead of a named or anonymous class.

Reply:
Done